src/controllers/user/update-user-details.controller.ts

```diff
@@ -0,0 +1,61 @@
+import { PrismaClient } from '@prisma/client';
+
+import User from '@/types/user-body-type';
+import asyncHandler from '../../utils/asyncHandler';
+import throwError from '../../utils/throw-error';
+
+const { user } = new PrismaClient();
+
+/**
+ * @auth required
+ * @route {POST} /user/update_user_details
+ * @body {name: string, description: string, phone: string, status: string}
+ * @returns {updated count}
+ */
+
+const updateUserDetails = asyncHandler(async (req, res) => {
+  try {
+    const { name, description, phone, status } = req.body as User;
+    const { id } = req.body as { id: string };
+
+    const body = {} as User;
```

Comment:
@Prodip-Kumar-Paul  Do not typecast an empty object add keys with "" values instead

Reply:
check below codes, my logic based on that scenario

how about using this ->  const body: Partial<User> = {};

Reply:
@Bijoy-007